features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -12,21 +12,24 @@ async function relevantZipEntries(zipFile) {
 }
 
 async function readJSONFile(relativeFileName, zipFile) {
-    const fullEntryName = `${zipFile.id}/${relativeFileName}`;
+    let fullEntryName;
+    // TODO: This is a hack to handle the fact that ids are
+    // different between the browser pod and the app one.
+    if (zipFile.id.startsWith("polypod://")) {
+        fullEntryName = `${zipFile.id}/${relativeFileName}`;
+    } else {
+        fullEntryName = `FeatureFiles/${zipFile.id}/${relativeFileName}`;
```

Comment:
Less than ideal, but right now it is a change localised here

Reply:
Hm, so if we already have the list of files - in the form of URLs, why do we need to _construct URLs_ here?

Reply:
Importers start from a know file name like `pages/pages_you_follow.json` and that one they need to transform into a fully qualified name like `FeatureFile/<uuid>/pages/pages_you_follow.json`

Reply:
I see - guess we can redesign that once we get the relative paths as part of the listing, but I suppose for now this is how it is.

One question though: Isn't this a fix that's completely unrelated to caching the listing? :D

I also did spot "Feature files" as an unknown folder in my report - possibly related?

Reply:
Yes related, `FeatureFiles` seems that will be an unknown file all the time now. We could make a bug report and squash that. Not sure exactly why but before we were assuming entries start with `polypod://`.

Let's say it is another improvement on top of caching the list :). Otherwise `readFullPathJSONFile` searches hundreds  of times for files instead of just accessing the file directly. That is particularly unfortunate, especially for reading message files where we already know full path.

Reply:
Got it!